docs/source/registrationOverview.rst

```diff
@@ -209,6 +211,17 @@ within each voxel, the latter uses the centers of all N voxels. Thus, for
 repeated random sampling with 100% rate, you get different samples and likely
 none of them is of the centers of all N voxels.
 
+Combining a mask with sampling is done using a rejection approach. First a sample is generated and
+then it is accepted or rejected if it is inside or outside the mask. This may cause
+problems when the mask region occupies a very small region in the original image. Because the sampling
+only discards data, the actual sample size is reduced. For some similarity metrics (e.g. mutual information)
```

Comment:
"the actual sample size is reduced" -> "the sample rate within the mask can be reduced from the requested"

Reply:
done